cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -265,7 +265,7 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) errors.AutoscalerError
 		glog.V(4).Infof("Calculating unneeded nodes")
 
 		scaleDown.CleanUp(time.Now())
-		managedNodes := getManagedNodes(autoscalingContext, allNodes)
+		managedNodes := getPotentiallyUnneededNodes(autoscalingContext, allNodes)
```

Comment:
nit: I guess it's not managed nodes any more. Can you update the name of the variable and the parameter of UpdateUnneededNodes method? Otherwise LGTM

Reply:
Done. Perhaps I've done a bit more renaming than strictly required, but hopefully variable names make more sense now.